packages/smart-contracts/scripts/deploy-payments.ts

```diff
@@ -328,12 +322,11 @@ export async function deployAllPaymentContracts(
     );
     addToResult(swapConversionResult);
 
-    // EthereumFeeProxy
-    const { address: ethFeeProxyAddress } = await runEasyDeployment({
-      contractName: 'EthereumFeeProxy',
-      artifact: ethereumFeeProxyArtifact,
-      nonceCondition: NONCE_BATCH_3 + 1,
-    });
+    // Compute EthereumFeeProxy address (CREATE2)
+    const ethFeeProxyAddress = await computeCreate2DeploymentAddress(
```

Comment:
This method logs a lot, which can possibly hide missing deployments. It would be nice to log less or only warnings when it's called in the context of live deployments. (although it's only a transition)